### PR TITLE
Miscellaneous geo fixes

### PR DESCRIPTION
**Pull request by etpinard**
Three geo fixes.  

The fix in https://github.com/plotly/plotly.js/commit/68bd47b754faac9e7a773f8d725444b1903bc08d is the most interesting where I spent most of the last two days trying to bring back my [`sane-topojson`](https://github.com/etpinard/sane-topojson) package back to life (a few things have changed in the [Natural Earth](https://www.naturalearthdata.com/) dataset and the [`topojson`](https://github.com/topojson/topojson) utility since 2016 :stuck_out_tongue_closed_eyes: ). I settled for a hacky but simple patch for now ... and until `sane-topojson` is back on solid grounds (cc https://github.com/etpinard/sane-topojson/pull/12)

@plotly/plotly_js reviewing this PR commit-by-commit should be straight-forward enough, but please let me know if you'd like a few codepen examples.

### CODE REVIEW

File: src/lib/geo_location_utils.js [hunk 1]
```diff
@@ -22,16 +20,53 @@ var locationmodeToIdFinder = {
...
+            // 'WA' is used for both Washington state and Western Australia.
+            // As subunits from USA and Canada never conflict, filtering out features
+            // south of the equator suffices to fix https://github.com/plotly/plotly.js/issues/3779
+            //
+            // A better fix would have us add a "governing unit" properties in subunit features
+            // in the `sane-topojson` package to avoid conflicts.
+            features2 = [];
+            for(i = 0; i < features.length; i++) {
+                f = features[i];
+                if(f.properties && f.properties.ct && f.properties.ct[1] > 0) {
```
Comment (archmoj) [inline src/lib/geo_location_utils.js]: The maximum latitude of Brazil is above 5N. It may be better to use greater than 6 here.

Comment (etpinard) [reply src/lib/geo_location_utils.js]: No conflicting _centroids_ go above the equator though.

**Comment by archmoj** [comment]
@etpinard Thanks for the fixes.
:dancer:

**Comment by etpinard** [comment]
Thanks for the review @archmoj 

I'll leave this PR open for now. I'll try to finish up https://github.com/etpinard/sane-topojson/pull/12 and make the fix in https://github.com/plotly/plotly.js/pull/3856/commits/68bd47b754faac9e7a773f8d725444b1903bc08d less hacky before 1.48.0 is ready.

**Comment by etpinard** [comment]
@archmoj I found time to :white_check_mark: https://github.com/etpinard/sane-topojson/pull/12 

This PR is `reviewable` again.

**Comment by archmoj** [comment]
@etpinard It would nice if [sane-topojson](https://github.com/etpinard/sane-topojson) could generate `dists` on multiple lines? Then it may be easier to compare the changes e.g. on the `GitHub`.

**Comment by etpinard** [comment]
> It would nice if [sane-topojson](https://github.com/etpinard/sane-topojson) could generate `dists` on multiple lines? Then it may be easier to compare the changes e.g. on the `GitHub`.

It could be nice. Feel free to open a new https://github.com/etpinard/sane-topojson/issues

But comparing MB-large json files isn't my thing, I comparing diff like in https://github.com/etpinard/sane-topojson/commit/8624ec41ab73e6d1b2be8e40478154b2dc3670b3 is much more valuable.

**Comment by archmoj** [comment]
Looking good.
Also thanks for [sane-topojson](https://github.com/etpinard/sane-topojson) upgrades.
:dancer:

**Comment by etpinard** [comment]
Thanks for the review and thanks for spotting the changes over Crimea:

Peek 2019-05-22 15-51

Peek 2019-05-22 15-52

I'll make sure to include a note about that in the changelog.

**Pull request merged**